repos/spack_repo/builtin/packages/libctl/package.py

```diff
@@ -27,6 +27,7 @@ class Libctl(AutotoolsPackage):
     )
 
     depends_on("c", type="build")  # generated
+    depends_on("fortran", type="build")  # generated
```

Comment:
@EMinsight does `libctl` fail to build for you without `fortran` or does it fail later to link with `meep`?